PyInstaller/building/splash.py

```diff
@@ -0,0 +1,531 @@
+            splash screen. This text will be used to show textual progress
+            in onefile mode.
+        :type text_pos: Tuple[int, int]
+        :keyword text_size:
+            The desired size of the font. If the size argument is a
+            positive number, it is interpreted as a size in points. If
+            size is a negative number, its absolute value is interpreted
+            as a size in pixels. Default ``12``
+        :type text_size: int
+        :keyword text_font:
+            An optional name of a font for the text. This font must be
+            installed on the user system, otherwise the system default
+            font is used. If this parameter is omitted, the default font
+            is also used.
+        :keyword text_color:
+            An optional color for the text. Either RGB HTML notation
+            or color names are supported. Default: black
+            (Windows: Due to a implementation issue the color magenta/
+            rgb(255, 0, 255) is forbidden)
+        :type text_color: str
+        :keyword text_default:
+            The default text which will be displayed before the extraction
+            starts. Default: "Initializing"
+        :type text_default: str
+        :keyword full_tk:
+            By default Splash bundles only the necessary files for the
+            splash screen (some tk components). This options enables
+            adding full tk and making it a requirement, meaning all tk
+            files will be unpacked before the splash screen can be started.
+            This is useful during development of the splash screen script.
+            Default: ``False``
+        :type full_tk: bool
+        :keyword minify_script:
+            The splash screen is created by executing an Tcl/Tk script.
+            This option enables minimizing the script, meaning removing
+            all non essential parts from the script. Default: True
+        :keyword rundir:
+            The folder name in which tcl/tk will be extracted at runtime.
+            There should be no matching folder in your application to
+            avoid conflicts. Default:  ``__splash``
+        :type rundir: str
+        :keyword name:
+            An optional alternative filename for the .res file. If
+            not specified, a name is generated.
+        :type name: str
+        :keyword script_name:
+            An optional alternative filename for the Tcl script, that
+            will be generated. If not specified, a name is generated.
+        :type script_name: str
+        :keyword max_img_size:
+            Maximum size of the splash screen image as a tuple. If the supplied
+            image exceeds this limit, it will be resized to fit the maximum
+            width (to keep the original aspect ratio). This option can be
+            disabled by setting it to None. Default: (760, 480)
+        :type script_name: Tuple[int, int]
```

Comment:
Copied and forgot to rename the variable. 

```suggestion
        :type max_img_size: Tuple[int, int]
```

Reply:
No CI isn't triggered by `[skip ci]`. AppVeyor and Travis respect that phrase - just put it before a blank line if you want AppVeyor to respect it. For some reason the following *isn't* respected by AppVeyor:
```
commit message

[skip ci]
```
But this is:
```
commit message
[skip ci]
```